.github/workflows/dispatch_change_dns_target_region.yml

```
@@ -0,0 +1,92 @@
+name: "[WD] Change DNS Target Region"
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to to update DNS target region for'
```

```suggestion
        description: 'Environment to update DNS target region for'
```

(as there's a follow up coming may be worth tackling in that)